src/scheduler.h

```diff
@@ -85,6 +85,9 @@ class Scheduler {
   // Run a new program. Returns the process ID of the root process.
   int run_program(Program* program, char** args, ProcessGroup* group, Block* initial_block);
 
+  // Run a new external program. Returns the process.
```

Comment:
External makes it sound a bit like we are fork-execing something.  Should we call it "native thread" instead?

Reply:
Yeah. Maybe we should tweak the names a bit.